scripts/scraper-ng/test/javascript-constructor.test.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs");
+const path = require("path");
+
+const { processFromSource, recipePath } = require("./utils");
+
+test.only("javascript-constructor", () => {
+  const file = processFromSource(
```

Comment:
If we think it is a common pattern to supply:
* HTML in a file
* a built-in recipe

...then we can make this a bit nicer for authors, by supporting `processFromFile` than takes a path, and by letting the recipe argument be any one of:
* recipe YAML
* a path to a YAML file (so we can supply custom recipes, but from a file)
* the name of a built-in recipe

Then this could be:

```js
const file = processFromFile(
  path.resolve(__dirname, "./javascript-constructor.html"),
  "javascript-constructor"
);
```

...which is easier to understand.

Reply:
Quite right. See 54c6d94f1536b9832f407843d3dc80b521f9472c for the refactor.